packages/gatsby/gatsby-plugin-clipanion-cli/gatsby-node.js

```diff
@@ -54,6 +54,18 @@ exports.sourceNodes = ({actions, createNodeId, createContentDigest}, opts) => {
         `\`\`\`\n`,
       ].join(``));
 
+      if (command.options.length > 0) {
+        sections.push([
+          `## Options\n`,
```

Comment:
```suggestion
          `## Extra options\n`,
```

Reply:
🤔 I'm not sure I understand this change - they aren't extra options, they *are* the options. The usage only includes positionals (and options without descriptions), while the details only include *some* options - the ones that have extra details. I don't think I understand how they are "Extra options" and not simply "Options". (Note that on the CLI command help page they are also shown as "Options" and not "Extra options")

Reply:
Yeah, if all the options are documented we can keep "Options" - since we support "partial option listing" (where only some options have description) I figured there might be some options that weren't worth formally documenting since they are well described in the details (like `-D` / `-P` on `yarn add`). No strong opinion.

Reply:
I've decided to document all options because:
1) It's more consistent.
2) It's useful to be able to read a single line and know what an option does instead of having to go through 10 paragraphs of "if this option is set, then [...]. Otherwise, if both of these flags are set, [...]. If this other option is set and the stars align, then [...]".

I think it would be best to keep it as "Options" unless there are any other reasons not to.